javascript/src/components/Modal.js

```diff
@@ -0,0 +1,121 @@
+                                </p>
+                                <p>
+                                    Dr. Brad St.Clair – Co-Principal Investigator<br/>
+                                    Research Geneticist, Pacific Northwest Research Station<br/>
+                                    USDA Forest Service, Corvallis, Oregon, USA<br/>
+                                    <a href="mailto:bstclair@fs.fed.us">bstclair@fs.fed.us</a>
+                                </p>
+                                <p>
+                                    Nikolas Stevenson-Molnar – Tool Developer<br/>
+                                    Software Engineer<br/>
+                                    Conservation Biology Institute, Corvallis, Oregon, USA<br/>
+                                    <a href="mailto:nik.molnar@consbio.org">nik.molnar@consbio.org</a>
+                                </p>
+                                <p>
+                                    Ken Ferschweiler – Tool Developer<br/>
+                                    Senior Software Architect, Modeler<br/>
+                                    Conservation Biology Institute, Corvallis, Oregon, USA<br/>
+                                    <a href="mailto:ken@consbio.org">ken@consbio.org</a>
+                                </p>
+                                <p>
+                                    Bill Klinkow – Tool Developer<br/>
+                                    Software Engineer<br/>
+                                    Conservation Biology Institute, Corvallis, Oregon, USA<br/>
+                                    <a href="mailto:bill.klinkow@consbio.org">bill.klinkow@consbio.org</a>
+                                </p>
+                                <p>
+                                    Brendan Ward – Project Manager<br/>
+                                    Conservation Biologist/GIS Analyst/Software Engineer<br/>
+                                    Conservation Biology Institute, Corvallis, Oregon, USA<br/>
+                                    <a href="mailto:bcward@consbio.org">bcward@consbio.org</a>
+                                </p>
+                            </div>
+                        </section>
+        }
+        return (
+            <div>
+                <div className="modal-background" onClick={() => this.props.onClose()}></div>
+                <div className="modal-card">
+                    <header className="modal-card-head">
+                        <p className="modal-card-title">People</p>
+                        <button
+                            className="delete"
+                            aria-label="close"
+                            onClick={() => this.props.onClose()}
+                        >
+                        </button>
+                    </header>
+                    {content[this.props.modalContent]}
```

Comment:
Consider passing content in rather than defining as part of `Modal` and then referencing id. E.g.,

```html
<Modal>
    <div>Modal content here</div>
</Modal>
```

Content is accessible through `this.props.children`.